apps/condo/domains/common/constants/countries.js

```diff
@@ -4,13 +4,25 @@ const DEFAULT_ENGLISH_COUNTRY = 'en'
 const RUSSIA_COUNTRY = 'ru'
 const COUNTRIES = {
     [DEFAULT_ENGLISH_COUNTRY]: {
-        adminRoleName: 'Admin',
+        roleNames: {
+            admin: 'Admin',
+            supervisor: 'Supervisor',
```

Comment:
To avoid misunderstanding issues, it seems, like `dispatcher` — is more consistent name with table, attached to the task.

Reply:
done